src/main/cpp/include/loader/genomicsdb_importer.h

```diff
@@ -119,6 +164,8 @@ class GenomicsDBImporter
     std::unordered_set<std::string> m_buffer_stream_names;
     VCF2TileDBLoader* m_loader_ptr;
     VCF2TileDBLoaderReadState* m_read_state;
+    const VidMapping *m_vid_map;
+    const CallsetMap *m_callset_map;
```

Comment:
Actually, I take that back - looks like you need these structures
However, you must handle them correctly in the move constructor and destructor